syntax = "proto3";


package a21;

service Server{
  
   
   rpc RegisterReplicaPrimary(Message) returns (Message) {}

}

message Message{
   string message = 1;
}